reflex/components/libs/chakra.py

```diff
@@ -121,6 +121,55 @@ class ChakraColorModeProvider(Component):
 LiteralVariant = Literal["solid", "subtle", "outline"]
 LiteralDividerVariant = Literal["solid", "dashed"]
 LiteralTheme = Literal["light", "dark"]
+LiteralCodeBlockTheme = Literal[
```

Comment:
can these be defined in `code.py`, since they're not really chakra-related and will be used even with radix.